apps/admin/src/components/notice/NoticePost/NoticePost.hooks.tsx

```diff
@@ -0,0 +1,12 @@
+import { useNoticePostMutation } from '@/services/notice/mutations';
+import { PostNoticeReq } from '@/types/notice/remote';
+
+export const useNoticePostAction = (noticeData: PostNoticeReq) => {
+    const { noticePostMutate } = useNoticePostMutation(noticeData);
+
+    const handlePostButtonClick = () => {
```

Comment:
```suggestion
    const handleNoticePostButtonClick = () => {
```
handlePostButtonClick은 너무 광범위합니다 handleNoticePostButtonClick으로 바꾸면 좋을것같아요

Reply:
넵